bcipy/gui/gui_main.py

```diff
@@ -142,7 +142,7 @@ def init_help(self, font_size: int, color: str) -> QWidget:
         """Initialize the help text widget."""
         if self.help_tip and self.label != self.help_tip:
             return static_text_control(None,
-                                       label=self.label,
+                                       label=self.help_tip,
```

Comment:
Thanks for fixing this. I noticed this locally and was going to add it to my next PR.